addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -650,6 +642,48 @@ public void execute(CommandLineArgument[] args) {
         }
     }
 
+    private void runPlanCommandLine(String source) {
+        if (source.startsWith("http")) {
```

Comment:
What is the source is `httpmyfile.yaml`?

Reply:
ohh, good one. Will rework to check first for a file and then fallback to URI.

Reply:
We could follow the approach as we are doing with the API add-ons: Attempt to parse as URL and fetch, otherwise try to read from the file system.